cache/disk/disk.go

```diff
@@ -645,6 +644,16 @@ func (c *diskCache) get(ctx context.Context, kind cache.EntryKind, hash string,
 		return nil, -1, nil
 	}
 
+	// Non-proxied Get requests do not seem to consume any significant amount of OS threads,
+	// and are therefore not throttled. However, it is assumed that proxied Get requests might,
+	// at least when storing the result from the proxy to disk, and perhaps also when
```

Comment:
I have been using all the other parts of #696 in production for long time. But not this part 4 commit for proxies, because I'm not using proxies myself.